core/domain/user_services_test.py

```diff
@@ -963,7 +964,11 @@ def test_last_logged_in_only_updated_if_enough_time_has_elapsed(self):
         # Without explicitly defining the type of the patched datetimes, NDB
         # validation checks for datetime.datetime instances fail.
         class PatchedDatetimeType(type):
+            """Validates the datetime instances for failure."""
```

Comment:
```suggestion
            """Validates the datetime instances."""
```

Validations need not always fail right?

Reply:
Done